src/components/ProfileBanner/Organization/index.js

```diff
@@ -8,7 +8,7 @@ import dp from "../../../assets/images/demoperson1.jpeg";
 import BannerDemo from "../../../assets/images/OrgBannerDemo.png";
 import iconbuttonImage from "../../../assets/images/unfilled3holes.svg";
 
-export default function Banner() {
+export default function Banner({ bannerImage, profileImage, name, story, followers, contributors, feed }) {
```

Comment:
did you use prettier in this line?

Reply:
I didn't. I have pushed changes after running prettier on all the files related to the pr.

Reply:
can you double check, the line is long and it's on a single line which should be broken down to new lines.

Reply:
I have made a new commit after running prettier. https://github.com/scorelab/Codelabz/commit/43d6a1e6a901cb728aa65b0923f189230941b2b9
Please check. This line has been broken to multiple short lines.